### PR TITLE
feat/QC-5371-removed-file

### PR DESCRIPTION
**Pull request by thomaskosciuch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,27 @@
...
+We executing a cron-job using [eventbridge](https://aws.amazon.com/eventbridge/) that triggers a [lambda](https://aws.amazon.com/lambda/) function and moves files from A to B using ssh file transfer protocols ([SFTP](https://en.wikipedia.org/wiki/SSH_File_Transfer_Protocol)).
+
+## SFTP
+The lambda is using [pysftp](https://docs.paramiko.org/en/latest/api/sftp.html#paramiko.sftp_client.SFTPClient.open) to create a connection, then I use the exposed the underlying [paramiko](https://docs.paramiko.org/en/latest/index.html) client (able to conduct more advanced operations) to read, and transfer the file.
+
+## Emails
+If the SFTP file transfer fails or succeeds emails are sent using [SES](https://aws.amazon.com/ses/)
+
+# Issues
+- Currently we are using welcome@qwealth.com instead of a not-stupid email
```
Comment (QW-Jordan) [inline README.md:21]: 💀

File: digitNbinTransfer/compare_files.py [hunk 2]
```diff
@@ -1,26 +1,26 @@
...
     destination_files: list[SFTPAttributes] = destination_connection.listdir_attr()
     destination_files: list[SFTPAttributes] = list(filter(lambda file: file.filename in source_file_array, destination_files))
     destination_file_times: dict[str, int] = {file.filename: file.st_mtime for file in destination_files}
     files_to_add: list[str] = []
-    
+
     def older_than(source_time:int , destination_time:int) -> bool:
         return source_time <= destination_time
 
-    for source_file, source_time in source_file_times.items():destination_connection
```
Comment (QW-Jordan) [inline digitNbinTransfer/compare_files.py:20]: oh this wouldn't work lol

File: digitNbinTransfer/failed_email.html [hunk 3]
```diff
@@ -350,6 +350,20 @@ <h1><a href="#">D1G1T SFTP TRANSFER</a></h1>
             				<h2><span>{n_transaction_files}</span> transactions files failed to upload to NBIN’s FTP folder at <span>{time}</span> on <span>{date}</span>.</h2>
             			</div>
             		</td>
+            	</tr>
+				<tr>
+            		<td>
+            			<div class="text" style="padding: 0 3em; text-align: center;">
+            				<h5>We are looking at your error as we're spying on when things go wrong.</h5>
```
Comment (QW-Jordan) [inline digitNbinTransfer/failed_email.html:357]: 🔎

File: README.md (hunk 1 above)
Comment (thomaskosciuch) [reply README.md:21]: f u lol i know

**Review by QW-Jordan — APPROVED** [review]

**Pull request merged**